src/System.Runtime.Serialization.Primitives/tests/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
+    </BuildConfigurations>
```

Comment:
why the tests are just built for netstandard ? why not netcoreapp, netfx also ?

Reply:
My understanding is you go for the most broad configuration (i.e. if there is nothing netfx/netcore specific you use netstandard) as the build tools with automatically pick up the right binary if you pass `/P:TargetGroup=netfx`